### ISSUE TITLE
/TOPICs longer than browser width do not wrap to a new line

### ISSUE THREAD

**Issue by W7PEA** [erming/shout#620]
1. Go to a channel with along topic, like say #shout-IRC 
2. Adjust your browser so it narrower than the topic is long

Observed:
Topic is truncated at the right margin

Expected:
Topic should wrap to a new line.

**Comment by MaxLeiter** [comment]
at the moment, the full topic is shown on hover, which im a fan of. The downside is theres no way to view the full topic on mobile, so I have mixed feelings on wrapping the title.

**Comment by YaManicKill** [comment]
Pretty sure they do wrap, we just only see 1 line. Not sure what the best idea for this is. Users can view the topic with the command, and I'm not convinced about showing multiple lines. Feels messy.

**Comment by W7PEA** [comment]
Frequently chanops put URLs in the /topic and you can't click them if you can't see them. 

Also, not everyone knows every single IRC command. Having it wrap, is more user friendly and inline with standard responsive web design.

**Comment by YaManicKill** [comment]
Yes, I understand your point entirely. But, my issue is that topics can be quite long, and to always wrap them onto multiple lines and show it all, all the time, wastes a lot of space for a lot of time. 

Having said that, I'm sure there would be another way we can do it (a button next to the topic to show the rest of the topic or something like that).

I'm not just saying leave it as it is.
